### PR TITLE
Insert Columns into clients (CU-86du6jp33)

### PR DESCRIPTION
**Pull request by elanawood**

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86du6jp33 Add "Organization", "Funder", "Zip/Postal Code" to Clients table. Fill in "Building Type"](https://app.clickup.com/t/86du6jp33)

**Review by micahdbak — COMMENTED** [review]
Same thing as the other PR!

**Comment by micahdbak** [comment]
Could you please run `npm audit fix` to fix the security vulns? Dependabot is too slow, just do it urself

**Comment by micahdbak** [comment]
After that I'm comfortable approving this!

File: server/db/046-insertColumns.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    migrationId := 46;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE clients_extension INSERT COLUMN organization default text;
```
Comment (micahdbak) [inline server/db/046-insertColumns.sql:15]: Please remove "default" from these

**Comment by micahdbak** [comment]
Pull from main - I think dependabot has fixed the security audits!!

File: server/db/046-insertColumns.sql (hunk 1 above)
Comment (elanawood) [reply server/db/046-insertColumns.sql]: done!

**Review by micahdbak — APPROVED** [review]
Looks good 👍

**Pull request merged**